packages/owlbot-bootstrapper/cli/cloudbuild-owlbot-bootstrapper.yaml

```diff
@@ -47,6 +53,12 @@ steps:
     - 'APP_INSTALLATION_ID=$_INSTALLATION_ID'
     - 'BUILD_ID=$BUILD_ID'
     - 'GITHUB_TOKEN=$_GITHUB_TOKEN'
+    - "--monoRepoPath"
+    - "$_MONO_REPO_PATH"
+    - "--serviceConfigPath"
+    - "$_SERVICE_CONFIG_PATH"
+    - "--interContainerVarsPath"
+    - "$_INTER_CONTAINER_VARS_PATH"
```

Comment:
These should be env vars, not CLI arguments.

Shouldn't the env var that the language container receive for SERVICE_CONFIG_PATH and INTER_CONTAINER_VARS_PATH be paths including the filename? I would not expect each container to have to know/specify the file name within the directory.